### ISSUE TITLE
[Proposal] Таймеры на выполнение целей некоторых антагов

### ISSUE THREAD

**Issue by EvilCoffe** [GammaStation/Gamma-Station#181]
Я предлагаю ввести ограниченное механом время на выполнение целей, так как ограничивать их правилами - бесполезно.
Суть заключается в чём, в данный момент существует проблема с мирномагами, трейторами и даже генокрадами, которая заключается в выжидании конца раунда, для беспалевного и безопасного выполнения целей, что просто превращает раунд в эксту, которая не нравиться тем, кто голосовал за секрет к примеру.  
Решить эту проблему я предлагаю введением таймеров на цели антагонистов, это стимулирует дрочеров на гринтекст выполнять цели не в конце раунда, когда все таймеры по сути кончаться и антаг проиграет, а в середине или даже в начале раунда. 
К примеру задание на кражу бака с фороном - максимально простое задание, которое выполняется любым трейтором за минут пять с начала раунда, должно иметь таймер минут на 20.
Таймер на чьё то убийство будет зависеть от профессии цели, чтобы дать антагу время. И всё в этом духе. 
Я думаю меня не одного достала эта проблема.

**Comment by UDaV73rus** [comment]
И вынашиваешь этот бак весь раунд, как настоящий профи шпион, не делая закладку?

**Comment by EvilCoffe** [comment]
Ну можно сделать два таких тригера на гринтекст. Первый - человек взял данный предмет за определённое время и второй тригер - человек улетел с этим предметом.И да, если ты к примеру инженер и у тебя задание на кражу магнабутсов или бака форона, то это уже хуйово будет работать, но тут уже цели надо для разных проф разные делать.

**Comment by RichardJones1** [comment]
Да, между прочим довольно логично не носить баки с собой а прятать их где-то. 

По поводу пропозала - соглашусь, хоть мое мнение может и не иметь особого веса. Как аргумент приведу правила с Поляриса:
"4.7 - Antagonist activities are to start before the transfer shuttle is called. Activities that are already in progress may be played out to their logical conclusion."

**Comment by WallShrabnic** [comment]
Полярис ЕМНИП мёртв

С другой стороны, это не будет работать, учитывая сложность и рандомность заданий (выпало украсть ядерный диск ассистухе при опытном капитане) 

С третьей - молчаливые роли, выполнившие цели в начале раунда и дрочащие остаток в работку тоже не нужны
